rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+
+    @book = book
+    book.rentals << self
+
+    @person = person
+    person.rentals << self
+  end
```

Comment:
- [ ] Kindly implement the suggestion I mentioned above book.rb `add_rental(book, date)`.